src/centralserver/admin-ui/src/components/managementRequests/ManagementRequestsList.vue

```diff
@@ -0,0 +1,290 @@
+        ></v-checkbox>
+      </div>
+    </div>
+
+    <v-data-table
+      :loading="loading"
+      :headers="headers"
+      :items="managementRequestsStore.items"
+      :search="filter.query"
+      :must-sort="true"
+      :items-per-page="10"
+      :options.sync="pagingSortingOptions"
+      :server-items-length="managementRequestsStore.pagingOptions.total_items"
+      class="elevation-0 data-table"
+      item-key="id"
+      :loader-height="2"
+      :footer-props="{ itemsPerPageOptions: [10, 25, 50] }"
+      @update:options="changeOptions"
+    >
+      <template #[`item.id`]="{ item }">
+        <div class="request-id">{{ item.id }}</div>
+      </template>
+
+      <template #[`item.created_at`]="{ item }">
+        <div>{{ item.created_at | formatDateTime }}</div>
+      </template>
+
+      <template #[`item.type`]="{ item }">
+        <type-cell :status="item.type" />
+      </template>
+
+      <template #[`item.security_server_owner`]="{ item }">
+        <div>{{ item.security_server_owner }}</div>
+      </template>
+
+      <template #[`item.security_server_id`]="{ item }">
+        <div>{{ item.security_server_id }}</div>
+      </template>
+
+      <template #[`item.status`]="{ item }">
+        <status-cell :status="item.status" />
+      </template>
+
+      <template #[`item.button`]="{ item }">
+        <div class="cs-table-actions-wrap management-requests-table">
+          <div v-if="item.status === 'WAITING'">
+            <xrd-button text :outlined="false">
+              {{ $t('action.approve') }}
+            </xrd-button>
+
+            <xrd-button text :outlined="false">
+              {{ $t('action.decline') }}
+            </xrd-button>
+          </div>
+        </div>
+      </template>
+    </v-data-table>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue, { PropType } from 'vue';
+import { RouteName } from '@/global';
+import StatusCell from '@/components/managementRequests/ManagementRequestStatusCell.vue';
+import TypeCell from '@/components/managementRequests/ManagementRequestTypeCell.vue';
+import { DataOptions, DataTableHeader } from 'vuetify';
+import { mapActions, mapState, mapStores } from 'pinia';
+import { notificationsStore } from '@/store/modules/notifications';
+import { debounce } from '@/util/helpers';
+import XrdFilter from '@/components/ui/XrdFilter.vue';
+import { managementRequestsStore } from '@/store/modules/managementRequestStore';
+import { userStore } from '@/store/modules/user';
+import { ManagementRequestsFilter } from '@/openapi-types';
+
+export enum Scope {
+  FULL,
+  SECURITY_SERVER,
+  MEMBER,
+}
+
+// To provide the Vue instance to debounce
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+let that: any;
+
+/**
+ * General component for Management requests
+ */
+export default Vue.extend({
+  name: 'ManagementRequestsList',
+  components: {
+    StatusCell,
+    TypeCell,
+    XrdFilter,
+  },
+  props: {
+    scope: {
```

Comment:
Do I understand correctly that this `scope` property currently has no effect whatsoever?
I.e. its usage will be implemented in separate user stories (member & securityserver detail views)

Reply:
Yes, same component will be used in another view. It will have a bit different behavior, but still same component.